src/templates/pageComponentsV2/LeftNav.jsx

```diff
@@ -2,12 +2,19 @@
 import PropTypes from "prop-types"
 import { useEffect, useState } from "react"
 
+import { useGetAllDirectoryPages } from "hooks/directoryHooks"
+
 import { generateLeftNav } from "templates/utils/leftnavGeneration"
 
-export const LeftNav = ({ dirData, fileName }) => {
+export const LeftNav = ({ siteName, fileName, collectionName }) => {
+  const params = {
+    siteName,
```

Comment:
Nit: we probably don't need `siteName` to be drilled here because we can retrieve it from 
`useParams()`

Reply:
Forgot that `useParams()` existed, fixed in bd527cb!